README.rst

```diff
@@ -90,25 +90,61 @@ Test Remote Server
 
 Integration tests can be executed on a remote server.
 
+To run the integration test suite, first build the integration tests container
+
 .. code-block:: shell
 
     docker-compose build tests
 
+or download a pre-built container from `Dockerhub <https://hub.docker.com/r/mozilla/remote-settings-integration-tests>`_.
+
+Next run the tests, supplying config values as necessary. Config values are
+set as environment variables provided to the Docker container. See
+``tests/conftest.py`` for descriptions of all of the config options that are
+available.
+
+Note that the tests assume that the server has the ``attachments``,
+``changes``, ``history``, and ``signer`` plugins enabled. It may optionally
+have the ``email`` plugin installed.
+
+To have the tests bootstrap themselves (i.e. when ``SKIP_SERVER_SETUP=false``),
+the credentials passed in ``SETUP_AUTH`` should have the permission to create 
+users, buckets, and collections.
+
+If the tests should not bootstrap themselves and instead use resources already
+available on the server (i.e. when ``SKIP_SERVER_SETUP=true``):
+
+- There should a bucket and collection available
+  
+  - the bucket, if not specified by the ``BUCKET`` config option, should be named ``main-workspace``
+  - the collection, if not specified by the ``COLLECTION`` config option, should be named ``product-integrity``
+
+- There should be two users available
+
+  - one user should be added to the ``editors`` group of the available collection
+  - the other should be added to the ``reviewers`` group of the available collection
+  - the credentials of these users should be passed in the ``EDITOR_AUTH`` and
+    ``REVIEWER_AUTH`` config options respectively
+
+Running integration tests on the Remote Settings dev server should look something like:
+
 .. code-block:: shell
 
-    docker-compose run \
+    docker run --rm \
         --env SERVER=https://settings.dev.mozaws.net/v1 \
-        --env MAIL_DIR="" `# disable tests about emails.` \
+        --env MAIL_DIR="" \
         --env SKIP_SERVER_SETUP=true \
-        --env EDITOR_AUTH=editor:azerty123 \
-        --env REVIEWER_AUTH=reviwer:s3cr3t \
-        tests integration-test
+        --env TO_REVIEW_ENABLED=false \
+        --env EDITOR_AUTH=<credentials available in 1Password> \
+        --env REVIEWER_AUTH=<credentials available in 1Password> \
```

Comment:
Shall we give a hint about format or username here?

```suggestion
        --env REVIEWER_AUTH=editor:<password available in 1Password> \
```

Reply:
[9dfaa00](https://github.com/mozilla/remote-settings/pull/227/commits/9dfaa0047a11a8ec109fda94c996ea9b794b753c)